app/src/main/java/ch/epfl/polybazaar/category/CategoryRepository.java

```diff
@@ -61,7 +83,13 @@ public static boolean contains(Category contained){
         multimedia.addSubCategory(new StringCategory("Computer"));
 
         Category videoGames = new StringCategory("Video games");
-        videoGames.addSubCategory(new StringCategory("Nintendo Switch"));
+        Category nintendoSwitch = new StringCategory("Nintendo Switch");
+        nintendoSwitch.addSubCategory(new StringCategory("Games"));
+        nintendoSwitch.addSubCategory(new StringCategory("Console"));
+        nintendoSwitch.addSubCategory(new StringCategory("Controllers"));
+        nintendoSwitch.addSubCategory(new StringCategory("Docks"));
+        nintendoSwitch.addSubCategory(new StringCategory("Accessories"));
+        videoGames.addSubCategory(nintendoSwitch);
```

Comment:
I think it would be interesting to be able to define categories in a more declarative way (in the database for example), this is probably out of the scope of this PR but we might want to keep this in mind for later.